deploy/sre-prometheus/ocm-agent/100-ocm-agent-upgrade.PrometheusRule.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-upgrade-managed-notification-alerts
+    role: alert-rules 
+  name: sre-upgrade-send-managed-notification-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-upgrade-managed-notification-alerts
+    rules:
+    - alert: UpgradeControlPlaneUpgradeTimeoutSRE
+      expr: avg_over_time(upgradeoperator_controlplane_timeout[10m]) == 1
```

Comment:
This same comment applies to all alerts defined here.

There will already be an alert with that name (the alert which notifies SRE), so this can't take the same name. It needs to be unique, ie something like `UpgradeControlPlaneUpgradeTimeoutNotificationSRE`

And whilst the alert condition is technically correct, I'm thinking the firing condition should be better based on the `UpgradeControlPlaneUpgradeTimeoutSRE` alert firing. That way if we ever change the implementation of the `UpgradeControlPlaneUpgradeTimeoutSRE` alert we don't need to remember to replicate it here.

And the severity can just be `info`